### DISCUSSION TITLE
G7 Missed Readings (Not Backfilling)

### DISCUSSION THREAD

**Discussion by yhonatanb20** [NightscoutFoundation/xDrip#3796] [Bug reports]
I've been experiencing this annoying issue of dexcom G7 readings (some 1-3 consecutive readings) just disappearing (as opposed to not being recieved and the later backfilled):

![Screenshot_20241205_072807_xDrip+](https://github.com/user-attachments/assets/ec1f2402-d868-42e5-b5c3-79ce8cac3af2)
![Screenshot_20241205_072756_xDrip+](https://github.com/user-attachments/assets/139b1335-9858-4ff1-aaff-8d735cc883b4)
![Screenshot_20241205_072747_xDrip+](https://github.com/user-attachments/assets/bb2b1e7d-4933-4ebf-a2e1-792710d2a019)

Using Samsung Galaxy a52s, latests nightly build and recommended settings.

**Comment by Navid200** [comment]
Backfill is working for you.  The readings that have a shade around them are backfills.
But, I have experienced a situation where I got Sensor error 7 for a while.  Later when the transmitter decided that everything was OK and resumed working, it did not provide any backfill.
Basically, if the transmitter decides that something is wrong with the sensor and it may need to be replaced, it stops providing readings.
This once happened to me when all of a sudden temperature dropped significantly while I was hiking.
After an hour when I was in the car driving hoe, readings returned.  But, I have a 1-hour gap.  

The next time this happens, if you temporarily switch to Dexcom app, you can see if Dexcom app backfills the same gap.  If it does, please let me know.  If it doesn't, it means that there is no data for that gap and transmitter did not consider the sensor properly operational in that period.

**Reply by yhonatanb20** [reply]
Thanks, this seems to happen quite often sometimes... You say "the transmitter considers the sensor", but on the G7 they are the same thing, I'm not sure I understand...

**Reply by Navid200** [reply]
I'm sorry about that.
I was referring to the functional aspects.  You are correct that physically there is only one element.

There is intelligence in the element.  It can stop providing readings if it suspects that something maybe wrong with the filament under your skin.

It's unusual if it happens frequently for no reason at all.
